### ISSUE TITLE
Reproduction and model correspondance to paper

### ISSUE THREAD

**Issue by SergeantChris** [LifengFan/Human-Gaze-Communication#4]
Very interesting work! Would you be so kind as to enhance your README with instructions on reproducing the experiments from the paper, as well as explanation on the source file names / abbreviations (e.g. how files in model dir correspond to different versions of your model). Thank you in advance

**Comment by hiraphor** [comment]
Agreed with @SergeantChris, the code does not run and makes many references to models and, pickle, and .npy files which are not included, as well as not being documented at all. Not useful for reproduction without some more explanation.
